src/components/Global/BreadCrumbs.vue

```diff
@@ -0,0 +1,133 @@
+<script>
+import { defineComponent } from "vue";
+import SectionContainer from "@/components/global/SectionContainer.vue";
+import BaseButton from "@/components/global/BaseButton.vue";
+import ArrowLeft from "@/assets/images/arrow_left.svg";
+import ChevronRight from "@/assets/images/chevron_right.svg";
+
+export default defineComponent({
+  name: "BreadCrumbs",
+  components: { SectionContainer, ArrowLeft, BaseButton, ChevronRight },
+  props: {
+    currentPageName: {
+      type: String,
+    },
+    parentPageName: {
+      type: String,
+    },
+    parentRouteName: {
+      type: Object,
+    },
+  },
+  methods: {
+    goBack() {
+      return this.$router.go(-1);
```

Comment:
It might be a good idea to also handle the case when there is no browser history